src/constants/data.tsx

```diff
@@ -1,169 +1,181 @@
-}
-
+    {
+      company: "Oregon State University",
+      title: "Undergraduate Teaching Assistant",
+      date: " • Dec 2021 - Dec 2023",
+    },
+    {
+      company: "Department of Veterans Affairs",
+      title: "Registered Dietitian",
+      date: " • Aug 2018 - Jun 2023",
+    },
+  ],
+  skills: {
+    languages: ["TypeScript", "Python", "Javascript", "Java", "HTML", "CSS"],
+    libraries: ["Node.js", "React", "Express.js", "Redux", "Flask"],
+    databases: ["MongoDB", "SQL", "DynamoDB"],
+    other: ["AWS", "Git", "Jira", "GraphQL", "Agile Development", "Rest APIs"],
+  },
+  education: [
+    {
+      school: "Oregon State University",
+      schoolInfo: "Bachelors of Science in Computer Science • ",
+      schoolDate: "Dec 2023 • GPA 3.9",
+    },
+    {
+      school: "University of Minnesota",
+      schoolInfo: "Bachelors of Science in Nutrition and Dietitics • ",
+      schoolDate: "Dec 2017 • GPA 3.3",
+    },
+  ],
+  email: "arsenaub@oregonstate.edu",
+  socials: [
+    { url: "https://www.linkedin.com/in/barsenault1/", icon: icons.linkedIn },
+    { url: "https://github.com/arse0039", icon: icons.github },
+  ],
+};
 
-export const davidProfile:Profile = {
-    // eslint-disable-next-line
-    intro: ["After 8 years in Business Operations as a Project Manager, People Manager, and Data Analyst, I embraced my passion for technology \
-    and problem-solving and returned to school to pursue a degree in Computer Science.", " Working in Business Ops \
+export const davidProfile: Profile = {
+  // eslint-disable-next-line
+  intro: [
+    "After 8 years in Business Operations as a Project Manager, People Manager, and Data Analyst, I embraced my passion for technology \
+    and problem-solving and returned to school to pursue a degree in Computer Science.",
+    " Working in Business Ops \
     I often had the opportunity to collaborate with software engineering teams. I always had an appreciation \
     for their contribution to the projects we worked on and thought it would be interesting to be on that side of the work,\
     actually ‘building’ the ideas that the business had come up with. Two years ago I started actively working \
     towards making the switch to a career as a Software Engineer by returning to school to pursue a degree in Computer \
-    Science at Oregon State University.", "Over the course of my degree program, I worked full-time, held a part-time position as a teaching \
+    Science at Oregon State University.",
+    "Over the course of my degree program, I worked full-time, held a part-time position as a teaching \
     assistant, and last summer I had a successful software engineering internship where I got professional experience working on a full-stack \
-    project. I returned to school following the intership and graduated in December.", "In addition to working on projects with Brandon, I'm now \
+    project. I returned to school following the internship and graduated in December.",
```

Comment:
Prettier formatted a lot of things but all I did was fix a type here.

Reply:
appreciate it!